lib/ArduinoNative/Arduino.cpp

```diff
@@ -335,18 +322,36 @@ static int handleCommandLineArguments(PrgArguments& prgArguments, int argc, char
             /* fallthrough */
 
         default: /* Default */
-            printf("Usage: %s <option(s)>\nOptions:\n", programName);
-            printf("\t-h\t\t\tShow this help message.\n");          /* Help */
-            printf("\t-p <PORT NUMBER>\tSet SocketServer port.\n"); /* Port */
-            printf("\t-n <NAME>\t\tSet instace name.");             /* Name */
             status = -1;
             break;
         }
 
         option = getopt(argc, argv, availableOptions);
     }
 
+    /* Does the user need help? */
+    if (0 > status)
+    {
+        printf("Usage: %s <option(s)>\nOptions:\n", programName);
+        printf("\t-h\t\t\tShow this help message.\n");                /* Help */
+        printf("\t-n <NAME>\t\tSet robot name.");                     /* Robot Name */
+        printf("\t-p <PORT NUMBER>\tSet SocketServer port.");         /* SocketServer Port */
+        printf(" Default: %s\n", PRG_ARG_SOCKET_SERVER_PORT_DEFAULT); /* SocketServer port default value*/
+    }
+
     return status;
 }
 
+/**
+ * Show program arguments on the console.
+ *
+ * @param[in] prgArgs   Program arguments
+ */
+static void showPrgArguments(const PrgArguments& prgArgs)
+{
+    printf("Robot name             : %s\n", prgArgs.robotName);
+    printf("SocketServer Port      : %s\n", prgArgs.socketServerPort);

```

Comment:
Remove spaces after "Port". The idea is that the longest line defines where the ":" is. The other lines will use spaces for filling up.